guides/get-started-microservices-on-kubernetes/simple-microservice-part2.adoc

```diff
@@ -0,0 +1,149 @@
+= {simple-microservice-part2}
+:summary: ONELINE EXPLANATION OF THE GUIDE
+:includedir: ../_includes
+include::{includedir}/_attributes.adoc[]
+include::./_includes/_titles.adoc[]
+// you can override any attributes eg to lengthen the
+// time to complete the guide
+:prerequisites-time: 10
+
+In this post, you will learn HOW-TO run the Docker Image you built in link:simple-microservice-part1[{simple-microservice-part1}] on Kubernetes.
+
+[[prerequisites]]
+== Prerequisites
+
+To complete this guide, you need:
+
+* Complete link:simple-microservice-part1[{simple-microservice-part1}]
+
+include::_includes/_constants.adoc[]
+
+== Image Registry
+
+To make the `{my-jaxrs-app-docker-image-name}:latest` Docker Image available to Kubernetes, you need to push it to some Image Registry that is accessible by the Kubernetes cluster you want to use;
+
+=== Quay.io
+
+There are many options to achieve this; in this post, we will push the `{my-jaxrs-app-docker-image-name}:latest` Docker Image, to the link:quay.io[quay.io] Image Registry;
+
+NOTE: You can skip this step in case you prefer to re-use the `quay.io/{quay-io-account-name}/{my-jaxrs-app-docker-image-name}` Docker Image, that we already pushed to link:quay.io[quay.io] in the `{quay-io-account-name}` account;
+
+Create a public repository named `{my-jaxrs-app-docker-image-name}` on link:quay.io[quay.io] (e.g. link:https://quay.io/repository/{quay-io-account-name}/my-jaxrs-app[https://quay.io/repository/{quay-io-account-name}/my-jaxrs-app]);
+
+NOTE: replace `{quay-io-account-name}` with the name of your account in all the commands that will follow
+
+Tag the Docker image:
+
+[source,bash,subs="normal"]
+----
+podman tag {my-jaxrs-app-docker-image-name} quay.io/{quay-io-account-name}/{my-jaxrs-app-docker-image-name}
+----
+
+Push the `{my-jaxrs-app-docker-image-name}` Docker Image to it:
+
+[source,bash,subs="normal"]
+----
+podman push quay.io/{quay-io-account-name}/{my-jaxrs-app-docker-image-name}
+----
+
+At this point, the `{my-jaxrs-app-docker-image-name}:latest` Docker Image should be publicly available and free to be consumed by any Kubernetes Cluster; you can verify this by running:
+
+[source,bash,subs="normal"]
+----
+podman pull quay.io/{quay-io-account-name}/{my-jaxrs-app-docker-image-name}
+----
+
+== Kubernetes
+
+=== Minikube
+
+You can use whatever Kubernetes cluster you have available; in this post, and in the following, we will use link:https://minikube.sigs.k8s.io/docs/[minikube];
+
+=== Deploy to Kubernetes
+
+To deploy our `quay.io/{quay-io-account-name}/{my-jaxrs-app-docker-image-name}` Docker image to Kubernetes, create a file named `deployment-{my-jaxrs-app-docker-image-name}.yaml` (see link:https://kubernetes.io/docs/concepts/workloads/controllers/deployment/[kubernetes deployment]) in the same directory as the `Dockerfile` and the `pom.xml` file, with the following content:
+
+.deployment-{my-jaxrs-app-docker-image-name}.yaml
+[source,yaml,subs="normal"]
```

Comment:
Plain yaml looks ok to me, but any reason for not using helm charts instead? Otherwise, you can also comment that helm charst can be used to automatize this deployment configuration

Reply:
@yersan I intentionally didn't use Helm Charts because I wanted to stick to the most basic approach ... thank you for the suggestion: I added a note that explicitly says that helm charts can be used